src/e2ee.rs

```diff
@@ -80,228 +76,221 @@ impl E2eeHelper {
         let public_key = match load_or_generate_self_public_key(context, &addr) {
             Err(err) => {
                 bail!("Failed to load own public key: {}", err);
-            } 
-            Ok(public_key) => public_key
+            }
+            Ok(public_key) => public_key,
```

Comment:
why not simply `let public_key = load_or_generate_self_public_key(context, &addr)?;`